### ISSUE TITLE
how to retrieve video trimmed from temporary directory?

### ISSUE THREAD

**Issue by kensou123** [sbis04/video_trimmer#18]
I want to load the output video file into my main page. Please suggest me

**Comment by sbis04** [comment]
You can just use the **path** of the output video file and pass it in the video player controller, like this:

```dart
final VideoPlayerController vcontroller = VideoPlayerController.file(File(videoPath));
```

**Comment by kensou123** [comment]
I use storageDir: StorageDir.temporaryDirectory. Does it do anything?

**Comment by sbis04** [comment]
Yes, you can use `StorageDir.temporaryDirectory`, it is a kind of cache directory and can be cleared at any time. 

You can get more info about the `StorageDir` types in the Docs: https://pub.dev/documentation/video_trimmer/latest/video_trimmer/Trimmer/saveTrimmedVideo.html

Okay, I understood you problem. There is no easy way to retrieve the output video path.

I will be deploying a fix pretty soon for that.

**Comment by sbis04** [comment]
Your issue should be fixed with the **v0.2.2**

Check the example app for more info.
